charts/csm-application-mobility/values.yaml

```diff
@@ -8,7 +8,7 @@ image:
   pullPolicy: IfNotPresent
 
 controller:
-  image: dellemc/csm-application-mobility-controller:v0.4.0
+  image: dellemc/csm-application-mobility-controller:v0.3.0
```

Comment:
Is this the latest?
I see v1.0.x tags in docker hub.
https://hub.docker.com/repository/docker/dellemc/csm-application-mobility-controller/general

Reply:
helm support for AM was dropped in versions after v0.3.0, so we need to keep v0.3.0 in helm charts from now on.